packages/commcare/src/Utils.js

```diff
@@ -53,8 +53,9 @@ export function request(configuration, path, opts) {
     headers,
     query: params,
     parseAs,
+    maxRedirections: 100,
```

Comment:
Ah yes,  can we set this right down to  `1`?  We shouldn't get getting redirected a lot and if we are, maybe it's only good  that we start seeing errors